.circleci/config.yml

```diff
@@ -30,8 +30,8 @@ commands:
       - run:
           name: Build
           command: |
-            go build
-            ./rainforest-cli -v --skip-update
+            go build -o rainforest
+            ./rainforest -v --skip-update
```

Comment:
do we still want these `--skip-update`s?

Reply:
I think so? The changes in this file are just so we build the same way in Circle as we do with GoReleaser and in the Dockerfile